src/components/catalog-search/catalog-search.html

```diff
@@ -1,5 +1,13 @@
-<div class="catalog-search">
-  <form role="form" class="landing-search-form search-pf has-button">
+<div class="catalog-search" ng-class="{'mobile-shown': $ctrl.mobileSearchInputShown}">
+  <a href=""
```

Comment:
If we use the `btn-link` class, can this switch to a button element without too much work?

For better accessibility and to avoid problems like https://bugzilla.redhat.com/show_bug.cgi?id=1470998, @jwforres wanted to try to switch to using buttons when we can for new code.

Reply:
Updated to a button. This shows a focus border which was a bit confusing to me, so I added some code to give focus to the search input when it is toggled open.